core/tests/protractor_utils/editor.js

```diff
@@ -176,31 +192,30 @@ var setInteraction = function(interactionId) {
     '.protractor-test-interaction-tab-' +
     INTERACTION_ID_TO_TAB_NAME[interactionId])).click();
   element(by.css('.protractor-test-interaction-tile-' + interactionId)).click();
+};
 
-  var elem = element(by.css('.protractor-test-interaction-editor'));
-  var customizationArgs = [elem];
+// This function is not usually necessary, please consider using
```

Comment:
This function should not usually be invoked directly; please consider using setInteraction instead.